src/DataTable/DataTable.svelte

```diff
@@ -231,7 +241,9 @@
                 dispatch('click:cell', cell);
               }}"
             >
-              <slot name="cell" row="{row}" cell="{cell}">{cell.value}</slot>
+              <slot name="cell" row="{row}" cell="{cell}">
+                {headers[j].display ? headers[j].display(cell.value) : cell.value}
```

Comment:
Should the `display` method be accessed this way—referencing the original `headers` array—or by passing it to the cells object?

https://github.com/IBM/carbon-components-svelte/blob/ead9634ab641ce50b13046604687164ee86be8cd/src/DataTable/DataTable.svelte#L119-L122

Reply:
What you have is good